package.json

```diff
@@ -51,7 +51,7 @@
     "login.dfe.winston-appinsights": "git+https://github.com/DFE-Digital/login.dfe.winston-appinsights.git#v3.0",
     "moment": "^2.29.1",
     "node-schedule": "^1.3.3",
-    "openid-client": "^1.20.0",
+    "openid-client": "^5.1.2",
```

Comment:
This seems like a significant version bump. Can we confirm that regression has been performed?